arcgis-ios-sdk-samples/Layers/Select ENC features/SelectENCFeaturesViewController.swift

```diff
@@ -0,0 +1,138 @@
+            mapView.touchDelegate = self
+            mapView.callout.isAccessoryButtonHidden = true
+        }
+    }
+    
+    /// The ENC layer that contains the current selected feature.
+    var currentENCLayer: AGSENCLayer?
+    /// A reference to the cancelable identify layer operation.
+    var identifyOperation: AGSCancelable?
+    
+    /// A URL to the temporary SENC data directory.
+    let temporaryURL: URL = {
+        let directoryURL = FileManager.default.temporaryDirectory.appendingPathComponent(ProcessInfo().globallyUniqueString)
+        // Create and return the full, unique URL to the temporary folder.
+        try? FileManager.default.createDirectory(at: directoryURL, withIntermediateDirectories: true)
+        return directoryURL
+    }()
+    
+    /// Load the ENC dataset and add it to the map view.
+    /// - Parameter mapView: The map view managed by the view controller.
+    func addENCExchangeSet(mapView: AGSMapView) {
+        let map = mapView.map!
+        // Load catalog file in ENC exchange set from bundle.
+        let catalogURL = Bundle.main.url(
+            forResource: "CATALOG",
+            withExtension: "031",
+            subdirectory: "ExchangeSetWithoutUpdates/ENC_ROOT"
+        )!
+        let encExchangeSet = AGSENCExchangeSet(fileURLs: [catalogURL])
+        
+        // URL to the "hydrography" data folder that contains the "S57DataDictionary.xml" file.
+        let hydrographyDirectory = Bundle.main.url(
+            forResource: "S57DataDictionary",
+            withExtension: "xml",
+            subdirectory: "hydrography"
+        )!
+        .deletingLastPathComponent()
+        // Set environment settings for loading the dataset.
+        let environmentSettings = AGSENCEnvironmentSettings.shared()
+        environmentSettings.resourceDirectory = hydrographyDirectory
+        // The SENC data directory is for temporarily storing generated files.
+        environmentSettings.sencDataDirectory = temporaryURL
+        // Update the display settings to make the chart less cluttered.
+        updateDisplaySettings(displaySettings: environmentSettings.displaySettings)
+        
+        encExchangeSet.load { [weak self] error in
+            guard let self = self else { return }
+            if let error = error {
+                self.presentAlert(error: error)
+            } else {
+                // Create a list of ENC layers from datasets.
+                let encLayers = encExchangeSet.datasets.map { AGSENCLayer(cell: AGSENCCell(dataset: $0)) }
+                // Add layers to the map.
+                map.operationalLayers.addObjects(from: encLayers)
+                mapView.setViewpoint(AGSViewpoint(latitude: -32.5, longitude: 60.95, scale: 1e5))
+            }
+        }
+    }
+    
+    /// Update the display settings to make the chart less cluttered.
+    func updateDisplaySettings(displaySettings: AGSENCDisplaySettings) {
```

Comment:
```suggestion
    func updateDisplaySettings(_ displaySettings: AGSENCDisplaySettings) {
```
Maybe worth add an underscore to make it read more like a sentence. 🤔